packages/editor/src/components/table-of-contents/style.scss

```diff
@@ -37,6 +37,12 @@
 	line-height: 30px;
 	color: $dark-gray-500;
 }
+.table-of-contents__popover .char-count {
```

Comment:
Since these styles just duplicate the styles above, we can include them above like so: 

```
.table-of-contents__number,
.table-of-contents__popover .word-count,
.table-of-contents__popover .char-count {
	font-size: 21px;
	font-weight: 400;
	line-height: 30px;
	color: $dark-gray-500;
}
```

We could also probably use the more generic `.table-of-contents__number` class name instead of a new `.char-count` here. I'm not sure why Word Count uses a unique class name there. 

We should also consider adding 8px ([`$grid-size`](https://github.com/WordPress/gutenberg/blob/08bf9e1adaef7c804c4715978e79cd4af2ac8f98/assets/stylesheets/_variables.scss#L19)) bottom margin to each `table-of-contents__count` item to add a bit more vertical space between the stacked items. (We'd also want to trim the `table-of-contents__counts` container by 8px to compensate for the extra space at the bottom.)

Reply:
Thanks for the suggestions regarding style changes. Do we also need to adjust the width of `table-of-contents__popover`, now its `min-width` is 380px. Initially, there are four columns in the popover so the width of each column is about 380/4. But now we only have three columns, so we can adjust the `min-width` to 380/4*3=285px.

Reply:
Thanks, @Jackie6! This looks great. Since the character count will likely be a longer number than the others, I think it makes sense to include a little more spacing here than there was before. I agree that the current `380px` is a little wide, but something like `320px` feels just right to me: 

<img width="388" alt="Screen Shot 2019-04-04 at 8 00 51 AM" src="https://user-images.githubusercontent.com/1202812/55554035-ccfe6380-56af-11e9-81bd-1c6056ed8373.png">

Let's adjust that — once that's in place, this is all set from a design perspective. ✅ 

Thank you for all your work here!